test/functional/feature_segwit.py

```diff
@@ -38,31 +38,29 @@ def find_spendable_utxo(node, min_value):
 
     raise AssertionError("Unspent output equal or higher than %s not found" % min_value)
 
+txs_mined = {} # txindex from txid to blockhash
```

Comment:
micronit: tiny preference for this being a member variable in `SegWitTest` rather than a global.